api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -43,4 +43,140 @@ object IgnoredParameters {
     case "H2ODeepLearning" => common ++ Seq("pretrained_autoencoder")
     case _ => common
   }
+
+  def ignoredInMOJOs(algorithm: String): Seq[String] = {
+    val common = manuallyImplementedInMOJOModel ++ unimplementedForMOJOs
+    common ++ {
+      algorithm match {
+        case "H2OGBMMOJOModel" =>
+          Seq("max_categorical_levels", "huber_alpha")
+        case "H2ODRFMOJOModel" =>
+          Seq("tweedie_power", "max_categorical_levels", "quantile_alpha", "huber_alpha", "custom_distribution_func")
+        case "H2OXGBoostMOJOModel" =>
+          Seq(
```

Comment:
@honzasterba Please could help me to identify parameters from the list that doesn't make any sense for XGBoost?

Reply:
please elaborate, parameters for what list doesnt make any sense for what?

Reply:
why not use XGBoostParametersV3.fields?

Reply:
@honzasterba That's a good point! I've exposed all fields with API notation from the class and parent classes. Just want to confirm, the `XGBoostParametersV3.fields`, `GBMParametersV3.fields`, `...` contains the list of relevant parameters for the particular algorithm and there are no exceptions. Is that correct?